js/data.js

```diff
@@ -7,15 +7,124 @@ var data = {
   nextEntryId: 1
 };
 
-var previousDataJSON = localStorage.getItem('javascript-local-storage');
+var previousDataJSON = localStorage.getItem('code-journal-local-storage');
 
 if (previousDataJSON !== null) {
   data = JSON.parse(previousDataJSON);
 }
 
 function handleStroage(event) {
   var dataJSON = JSON.stringify(data);
-  localStorage.setItem('javascript-local-storage', dataJSON);
+  localStorage.setItem('code-journal-local-storage', dataJSON);
 }
 
 window.addEventListener('beforeunload', handleStroage);
+
+var $entryForm = document.forms[0];
```

Comment:
This is dangerous if you ever added a new form to the page. I would select by a class or id on the `form` element